docs/docsite/rst/community/reporting_bugs_and_features.rst

```diff
@@ -16,35 +16,37 @@ Reporting a bug
 Security bugs
 -------------
 
-Ansible practices responsible disclosure - if this is a security-related bug, email `security@ansible.com <mailto:security@ansible.com>`_ instead of filing a ticket or posting to any public groups to receive a prompt response.
+Ansible practices responsible disclosure - for security-related bugs, email `security@ansible.com <mailto:security@ansible.com>`_ to receive a prompt response. Do not submit a  ticket or post to any public groups.
 
 Bugs in ansible-core
 --------------------
 
-Before reporting a bug, use the bug/issue search to check for `already reported issues <https://github.com/ansible/ansible/issues>`. Unsure if you found a bug? Report the behavior on the :ref:`mailing list or community chat first <communication>`.
+Before reporting a bug, use the GitHub issue search to check `already reported issues <https://github.com/ansible/ansible/issues>`. Unsure if you found a bug? Report the behavior on the :ref:`mailing list or community chat first <communication>`.
 
-Also, use the mailing list or chat if you are unsure whether a bug is in ``ansible-core`` or in a collection, and for "how do I do this" type questions to discuss.
+Also, use the mailing list or chat to discuss whether the problem is in ``ansible-core`` or a collection, and for "how do I do this" type questions.
 
-You need a free GitHub account to `report bugs <https://github.com/ansible/ansible/issues>`_ that affects
-- multiple plugins
-- a plugin that remained in the ansible/ansible repo
-- the overall functioning of Ansible
+You need a free GitHub account to `report bugs <https://github.com/ansible/ansible/issues>`_ that affects:
+
+- multiple plugins  
+- a plugin that remained in the ansible/ansible repo  
+- the overall functioning of Ansible  
 
 How to write a good bug report
 ------------------------------
 
-If you find a bug, open an issue using the `issue template <https://github.com/ansible/ansible/issues/new?assignees=&labels=&template=bug_report.yml>`. Detail what you've tried, why you think this is a bug, and what component to use. Fill it out as completely and as accurately as possible, include:
+If you find a bug, open an issue using the `issue template <https://github.com/ansible/ansible/issues/new?assignees=&labels=&template=bug_report.yml>`. 
```

Comment:
text instead of link 

```suggestion
If you find a bug, open an issue using the :ref:`issue template <https://github.com/ansible/ansible/issues/new?assignees=&labels=&template=bug_report.yml>`. 
```